pom.xml

```diff
@@ -27,6 +27,8 @@
     <!-- jhipster-needle-distribution-management -->
 
     <properties>
+        <!-- Swagger -->
+        <springfox.version>3.0.0-SNAPSHOT</springfox.version>
```

Comment:
do not put it at the beginning as it's less important than maven or java. Put it at the end plz